src/contexts/ConflictsContext.tsx

```diff
@@ -0,0 +1,13 @@
+import { Context, Dispatch, createContext, SetStateAction } from "react";
+
+interface ConflictsContextType {
+    conflictsSeverity: boolean;
```

Comment:
Although in the `CombinedProvider` you are passing a number as the severity, here on the context you are still saying that this will be a boolean.

You need to change the type of the variable in the interface and in the `createContext` call